lib/jetstream/pull_consumer.ex

```diff
@@ -192,23 +124,24 @@ defmodule Jetstream.PullConsumer do
   end
 
   def handle_continue(:connect, %{
-        connection_name: connection_name,
-        stream: stream,
-        consumer: consumer,
+        settings: %{
+          connection_name: connection_name,
+          stream: stream,
+          consumer: consumer
+        },
         module: module
       }) do
     with {:ok, conn} <- connection_pid(connection_name),
-         {:ok, _response} <- create_stream(conn, stream),
-         {:ok, _response} <- create_consumer(conn, consumer),
+         true <- Process.link(conn),
```

Comment:
```suggestion
         Process.link(conn),
```

should compile I guess?